officemanager/department/migrations/0003_auto_20190904_1324.py

```diff
@@ -0,0 +1,67 @@
+# Generated by Django 2.2.4 on 2019-09-04 11:06
+
+from django.db import migrations
+from random import randint, random
+
+def add_employees(apps, schema_editor):
+    
+    Emp = apps.get_model('department','Employees')
+    Dep = apps.get_model('department', 'Departments')
+    dep_list = Dep.objects.all()
+    fname_list = [
+        'Dmitri',
+        'ALexandra',
+        'Ivan',
+        'Anastasia',
+        'Lena',
+        'Krisitna',
+        'Nina',
+        'Madina',
+        'Yana',
+        'Kostya',
+        'Nikolay',
+        'Valeriy',
+        'Artem',
+        'Kirill',
+        'Daniil',
+        'Olga',
+        'Tamara'
+    ]
+    lname_list = [
+        'Divanko',
+        'Kaveruk',
+        'Korzh',
+        'Smit',
+        'Snow',
+        'Bolton',
+        'McFly',
+        'Alba',
+        'Stepaniuk',
+        'Kluni',
+        'Kot',
+        'Hyneman',
+        'Savage',
+        'Dronov',
+        'Lutz',
+        'Linus',
+        'Meladze'
+    ]
+    
+    for i in range(100):
+        emp = Emp(
+            department = dep_list[randint(0,len(dep_list)-1)],
+            first_name = fname_list[randint(0,len(fname_list)-1)],
+            last_name = lname_list[randint(0,len(lname_list)-1)],
+            d_of_b = '{}-{}-{}'.format(randint(1950,2010),randint(1,12),randint(1,28)),
+            salary = round(random()*10000,2)
+        )
+        emp.save()
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('department', '0002_auto_20190904_1052'),
+    ]
+
+    operations = [
+        migrations.RunPython(add_employees)
```

Comment:
Please add reverse_func in accordance with documentation: https://docs.djangoproject.com/en/2.2/ref/migration-operations/#runpython